extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json

```diff
@@ -65,7 +89,7 @@
                   "type": "object",
                   "properties": {
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForOpen": {
-                      "title": "Send Time Score for Open",
+                      "title": "Send Time Score for Open (to be deprecated)",
```

Comment:
Please add "meta:status": "deprecated" to the property to deprecate the property.